spark-warehouse/student/_delta_log/00000000000000000000.json

```diff
@@ -0,0 +1,3 @@
+{"commitInfo":{"timestamp":1725565486539,"operation":"WRITE","operationParameters":{"mode":"Overwrite","partitionBy":"[]"},"isolationLevel":"Serializable","isBlindAppend":false,"operationMetrics":{"numFiles":"1","numOutputRows":"0","numOutputBytes":"457"},"engineInfo":"Apache-Spark/3.5.0 Delta-Lake/3.1.0","txnId":"e3a74474-1ac4-4660-aa67-084f96dd16e2"}}
```

Comment:
Should this `spark-warehouse/student` files be uploaded? Seems like some test wrote them and weren't deleted

Reply:
That's a good catch. To be honest, I don't understand why we created this folder during the project test. Can we move it to tmp and make it delete it automatically?

Reply:
It shouldn't be created... All the tests should use the `tmpWarehouse` proposed by the `withSparkAndTmpWarehouse`. 
Probably you are using somewhere the `withSparkAndTmpDir` and it uses the current directory as a default table warehouse.

Reply:
Then it is best to delete the withSparkAndTmpDir and keep only the withSparkAndTmpWarehouse.